### ISSUE TITLE
[FileDropzone]: WCAG react issues

### ISSUE THREAD

**Issue by tobre6** [TEDI-Design-System/react#54]
## Problem
9.1.3.1 - Valitud failide sulgemisel ei saa aru, mida sulgetakse, sest ütleb ainult nupu nime.
9.2.1.1 - Disabled komponent ei ole päriselt disabled.
9.4.1.2 - Ei uuendata dropzone stataust: alati ütleb, et no file chosen.
9.4.1.3 - Ei loeta ette elemente, mida valitud on.
